src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -447,8 +447,9 @@ class DeployTask extends AbstractServerTask {
             Dependency dependency = entry.getValue();
             File dependencyFile = entry.getKey();
 
-            if (dependency instanceof ProjectDependency) {
-                Project dependencyProject = dependency.getDependencyProject()
+            if (dependency instanceof ProjectDependency) { 
+                def projectPath = dependency.getPath()
+                Project dependencyProject = task.project.findProject(projectPath)
```

Comment:
I noticed earlier in this method we use `task.getProject()` and not `task.project`. Is one way recommended over the other? Should we be consistent?